app.rb

```diff
@@ -1,10 +1,60 @@
 require 'sinatra'
 require 'active_record'
 require 'sqlite3'
+require 'pry'
 
 ActiveRecord::Base.establish_connection(
   adapter:  'sqlite3',
   database: 'db/development.sqlite3'
 )
 
+get "/api/notes/" do
+end
 
+get "/api/notes.json" do
+  notes = Note.all
+  output = notes.collect do |n|
+    {
+      "title" => n.title,
+      "body" => n.body,
+      "tags" => n.tags.collect { |t| {"name" => t.name} }
+    }
+  end
+  output.to_json(except: :id)
```

Comment:
Since you're building up a hash by hand, you likely don't need to specify the `except:`